srv/web/src/components/startingpages/StartingPage.tsx

```diff
@@ -17,18 +19,83 @@ export function Github() {
 window.location.href ='https://github.com/chronark/swapchain'
 }
 
-export default () => {
+/**
+ * This is a subset of all attributes from our mongo database.
+ *
+ * @interface Props
+ */
+export interface Props { 
+   address?: string, 
+   timelock?: number, 
+   txFees?: string, 
+   bidType?: string, 
+   askType?: string, 
+   bidAmount?: number, 
+   expiryDate?: string,
+   askAmount?: number
+}
+
+/**
+   * Create the handler that sends user accs input to our database.
+   *
+   *
+   *
+   */
+export default class Form extends Component<Props> { 
+   constructor(props: Props) {
+      super(props)
+
+      this.state = {
+         address: '',
+         timelock: '2 hours', 
+         txFees: '5000 Satoshis', 
+         bidType: 'BTS', 
+         askType: 'BTC', 
+         bidAmount: '', 
+         expiryDate: '',
+         askAmount: ''
+      }
+
+   }
+  
+changeHandler = (e: React.ChangeEvent<HTMLInputElement>): void => {
```

Comment:
Why don't you send the entire form data to the backend, rather than spamming your local state with change events?